programs/cleos/main.cpp

```diff
@@ -1725,6 +1752,86 @@ int main( int argc, char** argv ) {
    // create account
    auto createAccount = create_account_subcommand( create, true /*simple*/ );
 
+   // convert subcommand
+   auto convert = app.add_subcommand("convert", localized("Pack and unpack transactions"), false); // TODO also add converting action args based on abi from here ?
```

Comment:
To address your TODO, I do think having the ability to specify an abi file to use would be nice. In that case it should NOT connect to nodeos or need to.